packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -141,11 +145,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        with:
-          sparse-checkout: |
-            provisioning
-            tests
-            .config
```

Comment:
Did we remove this because it didn't give us much in a performance perspective, or because we were missing something?

Reply:
I think you can find the reason for removing this in the PR description.